plotly/offline/offline.py

```diff
@@ -142,8 +122,8 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly'):
     if show_link is False:
         link_text = ''
 
-    plotly_platform_url = py.get_config().get('plotly_domain',
-                                              'https://plot.ly')
+    plotly_platform_url = tools.get_config_file().get('plotly_domain',
```

Comment:
this should grab from `session` or else there can be a mismatch after a user sign-in. Just use `py.get_config()` if you can't grab session.